flags/tests/test_urls.py

```diff
@@ -226,14 +228,18 @@ def test_flagged_url_false_include_fallback_none(self):
             self.get_url_response("/include-fallback-include/included-url")
 
     @override_settings(FLAGS={"FLAGGED_URL": [("boolean", True)]})
-    def test_flagged_url_true_include_fallback_include_nonmatching_url(self,):
+    def test_flagged_url_true_include_fallback_include_nonmatching_url(
+        self,
+    ):
         with self.assertRaises(Http404):
             self.get_url_response(
                 "/include-fallback-include/other-included-url"
             )
 
     @override_settings(FLAGS={"FLAGGED_URL": [("boolean", False)]})
-    def test_flagged_url_false_include_fallback_include_nonmatching_url(self,):
+    def test_flagged_url_false_include_fallback_include_nonmatching_url(
+        self,
+    ):
```

Comment:
Would Black do this if there wasn't a trailing comma after `self`? (See also the `@override_settings` changes.) If not, I'd prefer to remove the trailing commas and keep them on one line. The primary benefit to me of the trailing commas is to reduce Git diffs when adding/removing items, which is irrelevant when the arguments still fit on one line.

Reply:
It looks like not. I'll amend that commit to remove any of those changes.

Reply:
TY!